### PR TITLE
Adds AUTHENTIC, JUICY, AMAZING BACON!

### PR DESCRIPTION
**Pull request by Chakirski**
Edited: 7/8/2016
-Adds ability to grill bacon from raw bacon after slicing a raw cutlet.

Sprite hacked and gutted from original telebacon. I have two actually, one with a tray and one without.
To remain as unbiased as possible, you look at the horror and tell me which, do you think, is better?

Tray - https://gyazo.com/e0662cfc7a5dcfe339847540476bb237
No Tray - https://gyazo.com/8abba5d31ce501dbd9dae82c0dc3ef8f

Bacon for both are the same size, though they might look slightly different in size.

:cl: Chakirski
imageadd: Adds new bacon sprites.
rscadd: Bacon can now be cooked from the grill. Just grill raw bacon.
tweak: Raw cutlets are now sliceable into raw bacon strips.
/:cl:

### CODE REVIEW

**Comment by Plotron** [comment]
Both look equally tasty.

**Comment by AceMcLazer** [comment]
I personally like the No-tray one. And, IMO two cutlets is quite a lot for bacon. I think maybe you could cut a cutlet into bacon and grill that for cooked bacon.. That would be cool.

**Comment by Chakirski** [comment]
I'm going to let this sit here for a day and see what the most popular response is.

**Comment by Fox-McCloud** [comment]
Needs a changelog

**Comment by Lady-Luck** [comment]
meme pr

**Comment by Plotron** [comment]
Bacon should be cheaper to make, I agree.

**Comment by Spacemanspark** [comment]
I don't like bacon.
Sue me.

**Comment by DaveTheHeadcrab** [comment]
Bacon is OK.
I do not get the cult worship that surrounds it.

**Comment by Fox-McCloud** [comment]
The sliceable types are largely meant for things like cakes, I don't think you need to repath them.

Just add an attackby for raw cutlets to get like 2-4 raw bacon that you can grill.

**Comment by Chakirski** [comment]
Everything that can be sliced is a sliceable, flat dough is also a sliceable. It just seems consistent, I don't know.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -2687,6 +2708,23 @@
...
+/obj/item/weapon/reagent_containers/food/snacks/rawcutlet
+	name = "raw cutlet"
+	desc = "A thin piece of raw meat."
+	icon = 'icons/obj/food/food_ingredients.dmi'
+	icon_state = "rawcutlet"
+	bitesize = 1
+	New()
+		..()
+		reagents.add_reagent("protein", 1)
+	attackby(obj/item/weapon/W as obj, mob/user as mob, params)
```
Comment (Fox-McCloud) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: attackby(obj/item/weapon/W, mob/user,  params)

Comment (tigercat2000) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: reeeeeeeeeee relative pathing

**Pull request merged**